client/app/src/main/java/com/example/mrides/Activity/ActivityObserver.java

```diff
@@ -1,8 +1,14 @@
 package com.example.mrides.Activity;
 
 
-
 public interface ActivityObserver {
 
+    /**
```

Comment:
Good addition of javadoc, this should be added to at least the critical methods in your code.